app/users/helpers.go

```diff
@@ -0,0 +1,94 @@
+package users
+
+import (
+	"bytes"
+	"net"
+	"net/http"
+	"strings"
+)
+
+// most of this is from https://husobee.github.io/golang/ip-address/2015/12/17/remote-ip-go.html
```

Comment:
i've seen this elsewhere, I think internal-apis.
perhaps it'd be time to unify this in lbry.go?

Reply:
Yeah, I copied it from internal-apis, which I cannot reuse in public code. One more strike and I'm moving it out.